internal/pkg/source/source.go

```diff
@@ -109,13 +109,23 @@ func (a *Source) visitOperands(operands []*ssa.Value) {
 		if !ok || a.marked[n] {
 			continue
 		}
-		al, ok := (*o).(*ssa.Alloc)
-		if !ok || al.Comment == "slicelit" || a.config.IsSource(al.Type()) {
+		al, isAlloc := (*o).(*ssa.Alloc)
```

Comment:
Please add some comments to explain the logic.

Reply:
Certainly, done. I think a comment is super relevant here. When adding the comment, I realized I could also refactor the code to be more in line with the "line of sight" idiom, and I think it reads much better now. Thanks!